samples/graph-chat-lifecycle/csharp/ChatLifecycle/Controllers/HomeController.cs

```diff
@@ -0,0 +1,304 @@
+﻿// <copyright file="HomeController.cs" company="Microsoft">
+// Copyright (c) Microsoft. All Rights Reserved.
+// </copyright>
+
+namespace ChatLifecycle.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using ChatLifecycle.Helper;
+    using ChatLifecycle.Models;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Graph;
+
+    public class HomeController : Controller
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private static readonly string pollyAppId = "AzureAd:PollyId";
+        public static string accessToken;
```

Comment:
@Trinetra-MSFT - If access token is kept in static field, this would cause overlap when two users are using the Tab at the same time. Could you please use session instead?